<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>domiciliaciones</class>
 <widget class="QDialog" name="domiciliaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesti√≥n domiciliaciones de vencimientos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="editarpushButton">
       <property name="text">
        <string>&amp;Editar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/boligrafo_editar.png</normaloff>:/iconos/graf/boligrafo_editar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nuevapushButton">
       <property name="text">
        <string>&amp;Nueva</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/Files-New-File-icon.png</normaloff>:/iconos/graf/Files-New-File-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="borrarpushButton">
       <property name="text">
        <string>&amp;Borrar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>editarpushButton</tabstop>
  <tabstop>nuevapushButton</tabstop>
  <tabstop>borrarpushButton</tabstop>
  <tabstop>salirpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>domiciliaciones</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
